src/server-code/admin-server.js

```diff
@@ -141,23 +140,20 @@ export default function runAdminServer(serverFalcorModel) {
 
   app.post('/upload', upload.single('image'), (req, res) => {
     const filePath = req.file.path;
-    const deleteTmpFile = () => {
-      fs.unlink(filePath, (err) => {
-        if (err) {
-          console.error(err); // eslint-disable-line no-console
-        }
-      });
-    };
+
     if (isDevelopment) {
       /**
        * As we are in dev-mode, we don't actually want to upload to s3.
        * You can either compile with production mode or remove this
        * temporarily if extra s3 tests are needed at some point.
        */
-      setTimeout(() => {
-        res.status(200).send('success test_url');
-      }, 2000);
-      deleteTmpFile();
+      compressJPEG(filePath).then(() => {
+        setTimeout(() => {
+          res.status(200).send('success on test url');
```

Comment:
This is my bad from changing this because I forgot the functionality of my own code during linting fixes. The "API" is `success ${url}` so right now the front-end would show `URL: on` so if we could just change this back to what it was before I fixed it which I fixed in a different branch but before you branched (again, my bad not yours!) to `success test_url`